lib/Conversion/TritonGPUToLLVM/ConvertLayoutOpToLLVM.cpp

```diff
@@ -276,11 +276,6 @@ struct ConvertLayoutOpConversion
     unsigned outVec = 0;
     auto origRepShape = getRepShapeForCvtLayout(op);
     auto paddedRepShape = getScratchConfigForCvtLayout(op, inVec, outVec);
-    if (getElementTypeOrSelf(op.getType())
-            .isa<mlir::Float8E4M3B11FNUZType, mlir::Float8E4M3FNType>()) {
```

Comment:
I'm confused why we remove the branch entirely; isn't Float8E4M3FNType still used?

Reply:
no it is removed, added some clarification in description